codec-http/src/main/java/io/netty5/handler/codec/http/headers/DefaultHttpSetCookie.java

```diff
@@ -196,28 +202,35 @@ static HttpSetCookie parseSetCookie(final CharSequence setCookieString, boolean
                         if (isWrapped) {
                             parseState = ParseState.Unknown;
                             value = setCookieString.subSequence(begin, i);
-                            // Increment by 3 because we are skipping DQUOTE SEMI SP
-                            i += 3;
+                            if (validateContent) {
+                                // Increment by 3 because we are skipping DQUOTE SEMI SP
```

Comment:
Can you link to the RFC ?

Reply:
It's https://www.rfc-editor.org/rfc/rfc6265#section-4.1.1
It'll add this to the comment.